### ISSUE TITLE
Admin sees GitHub returning 404 when "Edit"ing environments

### ISSUE THREAD

**Issue by bkuhl** [rancher/rancher#6955]
As an administrator for Rancher, sometimes when I go to `Manage Environments` and then choose `Edit` on certain environments I'm presented with this error:

<img width="521" alt="screen shot 2016-12-06 at 10 39 48 am" src="https://cloud.githubusercontent.com/assets/524933/20934464/44c1b9d4-bba9-11e6-8493-c94a6e4306b3.png">

This doesn't happen on all environments, just some.  However it consistently happens on those some.  What steps can I take to further debug this issue?  The audit log contains no details.

---
| Useful | Info |
| :-- | :-- |
|Versions|Rancher `v1.1.4` Cattle: `v0.165.8.4` UI: `v1.1.14` |
|Access|`github` `admin`|
|Orchestration|`Cattle`|
|Route|`settings.projects.index`|

**Comment by ebr** [comment]
Confirming this... Can't edit any of my environments. Same Rancher/Cattle versions as @bkuhl. I am `owner` in both Github and Rancher.

In browser console logs:`GET https://rancher.my.domain/v1/projects/1a7/projectmembers 503 (Service Unavailable)`

Confirmed via the API: `https://rancher.my.domain/v1/projects/1a7/projectmembers` returns:
```
{
"id": "94381685-0204-4652-98b9-91e300d4b074",
"type": "error",
"links": { },
"actions": { },
"status": 503,
"code": "GitHubError",
"message": "Non-200 Response from Github",
"detail": "Status code from Github: 404",
}
```

**Comment by ebr** [comment]
@bkuhl i believe this is the reason. https://github.com/rancher/rancher/issues/5801. The ACL contains someone who was removed from the GitHub org.

Now i'm just trying to find a workaround, because upgrading Rancher is not an option at this moment in time.

**Comment by ebr** [comment]
I was able to get this fixed by editing the `project_member` table in the database, and setting the offending (non-existing) members' `state` to `purged`

**Comment by bkuhl** [comment]
Interesting.  Thanks for pointing that out.

**Comment by aemneina** [comment]
Looks like this was fixed in 1.3 and workaround seems to work for versions prior. Closing issue.

**Comment by ecliptik** [comment]
Looks like this is happening again in 1.5.6.

1. Github Authentication was enabled
2. De-activate Authentication
3. Enabled Active Directory Authentication
4. Can edit "default" environment, but all other environments show up as blank when trying to edit them

<img width="1427" alt="screen shot 2017-05-17 at 10 30 00 pm" src="https://cloud.githubusercontent.com/assets/1261742/26187820/686e31f6-3b50-11e7-8617-2b2703196d92.png">

My account is an owner in Access Control and should have full access to all environments.
